package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "code_compete",
+  "version": "1.0.0",
+  "description": "Coding Competition with TDD and TypeScript. \"Code Compete\" team will use this repository for practicing best practices while solving algorithmic challenges. Every day, a new challenge will be introduced to the team, team has to solve the challenge and commit their solution to this repository by making a PR to the master branch.",
+  "main": "jest.config.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.1.0",
+    "@typescript-eslint/parser": "^4.1.0",
+    "eslint": "^7.8.1",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "husky": "^4.3.0",
+    "jest": "^26.4.2",
+    "prettier": "^2.1.1",
+    "ts-jest": "^26.3.0",
+    "typescript": "^4.0.2"
+  },
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "prettier:format": "prettier --config .prettierrc 'src/**/*.ts' --write"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm test",
+      "pre-push": "npm run test"
```

Comment:
Can we put this instead here?
```
      "pre-commit": "yarn lint:fix && yarn test",
      "pre-push": "yarn test"
```

Reply:
Yes we can. However, I would want this to done in separate issue. Will create and assign the issue to you.

Reply:
ok. I will raise pr there.

Reply:
No. I have created and assigned you a ticket. You should apply the changes in that ticket.